src/pages/contact_information.tsx

```diff
@@ -55,36 +57,59 @@ const contactCards = [
 	},
 ];
 
-const ContactCard = ({ title, operationTime, operationTitle, imgUrl, email }: ContactCardProps) => (
-	<Card className="mx-auto max-w-xs rounded-lg p-10 md:p-10 lg:p-10">
-		<CardSection className="max-w-xs">
-			<Image
-				component={NextImage}
-				className="h-full w-full rounded-lg"
-				src={imgUrl}
-				alt="Stamford Logo"
-				width={384}
-				height={216}
-			/>
-		</CardSection>
+const ContactCard = ({ title, operationTime, operationTitle, imgUrl, email }: ContactCardProps) => {
+	const [isCopied, setIsCopied] = useState(false);
 
-		<CardSection className="font-open-sans mt-4 flex text-lg font-semibold text-white md:mt-6 lg:mt-8">
-			{title}
-		</CardSection>
+	const handleCopyToClipboard = async () => {
+		try {
+			await navigator.clipboard.writeText(email);
+			setIsCopied(true);
 
-		<CardSection className="mt-2 flex items-center text-sm md:mt-4 md:text-base lg:mt-6">
-			{operationTitle}
-			<br />
-			{operationTime}
-		</CardSection>
+			setTimeout(() => {
+				setIsCopied(false);
+			}, 1500);
+		} catch (error) {
+			console.error("Unable to copy to clipboard", error);
```

Comment:
Do we inform the user if the email cannot be copied? Just so they're aware of it

but tbh I think it' a very rare case

Reply:
hmm. we need tawan's suggestion on this.

Reply:
@Lxkas

Reply:
This is a pretty critical error if it happens, ensure that it throws, not just error so that it gets logged to Sentry, be sure to also notify the user since there is something seriously wrong if you can't access the clipboard in this context xd

Reply:
Waaait a second.. why are you not using Mantine's hooks for this?

Reply:
or we can just say copy failed in a btn with red backgroud. in a place for copied? i guess

Reply:
And:
https://mantine.dev/core/copy-button/

Reply:
can we assume that there will be no copy error if we use this mantine component? if so then let's remove this try-catch block completely

Reply:
i am new with mantine :3 excuse me

Reply:
> can we assume that there will be no copy error if we use this mantine component/hook? if so then let's remove this try-catch block completely

Yeah, can assume so

Reply:
> i am new with mantine :3 excuse me

That's OK, it is what reviews are for :>